### ISSUE TITLE
Increase clickable area of action button in Card components

### ISSUE THREAD

**Issue by suevalov** [contentful/forma-36#416]
![Screenshot 2020-02-11 at 10 11 55](https://user-images.githubusercontent.com/3672221/74236365-57045900-4cd1-11ea-8075-98eef75d634b.png)

I would be nice to increase a clickable area of this button to improve usability and a11y. As for me, that click at right right corner should always trigger an action. Right now I miss almost every second time.

**Comment by mshaaban0** [comment]
@suevalov Thanks for reporting. More info about the target size (clickable area) guidelines from [WCAG Standards](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html)

> Ensuring that touch targets are at least 44 by 44 CSS pixels.

**Comment by m10l** [comment]
@mshaaban0 These recommendations are for touch screen devices, not something we target.

**Comment by mshaaban0** [comment]
@m10l Maybe we should target it :) doing this will fix it for mouse, and touch screens. The guidelines however, are for both.

> Pointer input:
input device that can target a specific coordinate (or set of coordinates) on a screen, such as a mouse, pen, or touch contact

**Comment by m10l** [comment]
@mshaaban0 Ah my bad! I misinterpreted the document.

**Comment by mshaaban0** [comment]
`CardActions` is using `IconButton` component, shall I reflect the change on the `IconButton` component and apply it everywhere, or scope it to `CardActions`

IconButton is used in:
- CardActions
- ModalHeader
- Workbench
- NotificationItem
- EntityListItem
- EditorToolbarButton
- AutoComplete

**cc:** @m10l @wichniowski @burakukula

**Comment by burakukula** [comment]
@mshaaban0 I guess that it would be good to solve this issue globally, it would be beneficial for all the components that are using IconButton. WDYT?

**Comment by suevalov** [comment]
is there any progress on this?

**Comment by joneskj55** [comment]
@suevalov Is this still something you all are considering? If so, I'd like to tackle it.

**Comment by suevalov** [comment]
@joneskj55 Yep, I think it would be a valuable addition.

@m10l  @denkristoffer what do you think?

**Comment by denkristoffer** [comment]
I'm all for it 👍

**Comment by joneskj55** [comment]
Cool. I'm looking through https://f36.contentful.com/ in order to try and reproduce this and cannot find the three-dot button shown in your original comment. It is likely that I am on the wrong site. Could you please point me in the right direction?

**Comment by denkristoffer** [comment]
@joneskj55, sorry about the confusing websites, we're working on it. You'll want to look in [the Storybook here](https://f36-storybook.contentful.com/?path=/story/components-card-cardactions--default).

**Comment by joneskj55** [comment]
Ok thanks, got it. Will submit PR shortly.

**Comment by gui-santos** [comment]
I'm including this in v4.0

The reason can be found in the PR linked to this issue, but basically, we found out that increasing the clicking area of the IconButton would be a breaking change

**Comment by suevalov** [comment]
It's finally out in Forma v4 release. Closing the issue.
